.github/ISSUE_TEMPLATE/orientation-epic.md

```diff
@@ -14,30 +14,43 @@ As a new VA.gov team member, I need to complete onboarding activities so I can s
 - [ ] Each team member has completed each item in the orientation checklist below.
 
 ## Onboarding Checklist
+### Share your info with others
+- [ ] Name
+- [ ] Role
+- [ ] Email
+- [ ] Github handle
+- [ ] Start date
+- [ ] VFS Team
 
+### Attend general onboarding
```

Comment:
LOVE the new organization!!!

Reply:
Thanks! Can we add preferred pronouns to the info list so we can head off any embarrassment if gender noncomforming folk join up?

Reply:
Yes, that's a great idea.!